requirements.txt

```diff
@@ -3,3 +3,4 @@ cld2_cffi~=0.1
 spacy~=2.0
 textacy~=0.6
 textpipe-pattern==3.6
+msgpack-numpy==0.4.4
```

Comment:
not related to this change, but msgpack-numpy released 0.4.4.1 which broke our build.

Reply:
Maybe you can add a comment in the req file (not sure if that is possible)? We can drop this dependency in the future.